implementations/rust/ockam/ockam_identity/src/credential.rs

```diff
@@ -64,9 +68,35 @@ impl fmt::Display for Credential {
             " Issuer: {} ({})",
             credential_data.issuer, credential_data.issuer_key_label
         )?;
-        //TODO: write timestamps on human-readable format. Should we add a dependency for this?
-        writeln!(f, " Created: {}", u64::from(credential_data.created))?;
-        writeln!(f, " Expires: {}", u64::from(credential_data.expires))?;
+        
+        let human_readable_time = |time: Timestamp| {
```

Comment:
That's great thanks! What would be even better would be a unit test for this `Display` instance if you have a bit of time :-)

Reply:
let me try to refactor according to what you have mentioned in bonus points, then i will write unit test

Reply:
Thank you